_projects/go_bears_on_bsd.md

```diff
@@ -1,7 +1,8 @@
 ---
 collaborating_projects:
   - coala
-desc: "Explore improvements to coala's dependency management through having Go Linters installed in *BSD systems automatically."
+desc: "Explore improvements to coala's dependency management through having Go Linters installed in *BSD systems 
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.